### PR TITLE
bugfix: fix mysql insert use select last insert id is undo_log id value

### PR DESCRIPTION
**Pull request by jsbxyyx**
### Ⅰ. Describe what this PR did
mysql business insert use select last insert id is undo_log id's value.

### Ⅱ. Does this pull request fix one issue?

fixes: #2655

### CODE REVIEW

**Comment by a364176773** [comment]
is mysql the only one that need to handle ?

**Comment by jsbxyyx** [comment]
@a364176773 oracle, postgresql use sequence.

File: script/client/at/db/mysql.sql [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
     `branch_id`     BIGINT(20)   NOT NULL COMMENT 'branch transaction id',
     `xid`           VARCHAR(100) NOT NULL COMMENT 'global transaction id',
     `context`       VARCHAR(128) NOT NULL COMMENT 'undo_log context,such as serialization',
     `rollback_info` LONGBLOB     NOT NULL COMMENT 'rollback info',
     `log_status`    INT(11)      NOT NULL COMMENT '0:normal status,1:defense status',
-    `log_created`   DATETIME     NOT NULL COMMENT 'create datetime',
-    `log_modified`  DATETIME     NOT NULL COMMENT 'modify datetime',
-    PRIMARY KEY (`id`),
-    UNIQUE KEY `ux_undo_log` (`xid`, `branch_id`)
+    `log_created`   DATETIME(6)  NOT NULL COMMENT 'create datetime',
```
Comment (slievrly) [inline script/client/at/db/mysql.sql:9]: Where is used the precision of time compared?

Comment (jsbxyyx) [reply script/client/at/db/mysql.sql:9]: the log_created and log_modified field precision not use in current version. I think the datetime necessary precision to millisecond above.

Comment (slievrly) [reply script/client/at/db/mysql.sql:9]: If that code needs to be changed to the same precision, now(6)

Comment (jsbxyyx) [reply script/client/at/db/mysql.sql:9]: OK

**Pull request merged**